pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/operator/WindowAggregateOperator.java

```diff
@@ -239,16 +238,18 @@ private boolean consumeInputBlocks() {
         return true;
       }
 
+      Key emptyOrderKey = AggregationUtils.extractEmptyKey();
```

Comment:
i dont think we need to extract this on every block

Reply:
good catch, fixed